### ISSUE TITLE
insertEntity: change the validation of PartitionKey and RowKey

### ISSUE THREAD

**Issue by fcatae** [Azure/azure-storage-node#154]
I wanted to use plain JSON objects with no need to use 'entityGenerator'. However, it does not work and it fails during a argument validation (ArgumentValidator).

[Simple example](http://dl.windowsazure.com/nodedocs/)

```
var tableService = azure.createTableService(),
    task1 = {
        PartitionKey : 'tasksSeattle',
        RowKey: '1',
        Description: 'Take out the trash',
        DueDate: new Date(2011, 12, 14, 12)
    };
tableService.insertEntity('tasktable', task1, function(error){
    if(!error){
        // Entity inserted
    }
});
```

An easy fix is to change PartitionKey and RowKey values:

        PartitionKey : { '_': 'tasksSeattle'},
        RowKey: { '_': '1'},

and then it works.

A better solution would be accepting PartitionKey and RowKey as strings by either:
1) implicitly convert the fields PartitionKey and RowKey into objects
2) change the validation of PartitionKey and RowKey to accept strings

**Comment by hasonmsft** [comment]
Thank @fcatae for the valuable suggestion. We can add this in next release. Will get you updated when it's ready.

**Comment by vinjiang** [comment]
It's now fixed in 1.1.0.
